css/app.css

```diff
@@ -35,3 +35,28 @@ gaia-list .destructive {
 a.navigation *, a.action *, a.edit * {
   pointer-events: none;
 }
+
+gaia-list span.group {
+  text-transform: capitalize;
+  font-size: 0.8em;
+  min-height: auto;
+
+  background: rgba(255, 255, 255, 0.35);
+  color: var(--text-color);
+}
+
+gaia-list span.group::before {
+  position: absolute;
+  top: 0;
+  left: -20px;
+  width: calc(100% + 20px);
+  height: 100%;
+  content: '';
+
+  background-color: var(--highlight-color);
+  opacity: 0.2;
+}
```

Comment:
and actually this does not seem to work for me... maybe that's why I'd like a bigger font, maybe we don't need it if this works?

I tried only in Firefox though, not on the device...